### PR TITLE
add h&m

### PR DESCRIPTION
**Pull request by Lightning5trike**

### CODE REVIEW

File: entries/h/hm.com.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "H & M": {
+    "domain": "hm.com",
+    "url": "https://www.hm.com/",
```
Comment (ApCoder123) [inline entries/h/hm.com.json]:
```suggestion
```
The `url` field is not necessary in this case, as the site is accessible from https://`domain`

File: entries/h/hm.com.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "H & M": {
```
Comment (ApCoder123) [inline entries/h/hm.com.json]:
```suggestion
  "H&amp;M": {
```

**Review by ApCoder123 — CHANGES_REQUESTED** [review]
Please use this SVG copy of the logo instead of PNG.

File: entries/h/hm.com.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "H&amp;M": {
+    "domain": "hm.com",
+    "img": "hm.com.png",
```
Comment (ApCoder123) [inline entries/h/hm.com.json]:
```suggestion
```
The `img` field is not necessary when using SVGs

**Review by ApCoder123 — DISMISSED** [review]

**Review by ApCoder123 — CHANGES_REQUESTED** [review]
Please remove the `img` field

**Review by ApCoder123 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**